static/src/javascripts/projects/common/modules/experiments/tests/contributions-moment-banner-environment.js

```diff
@@ -34,6 +34,14 @@ const minArticlesBeforeShowingBanner = 0;
 const linkUrl =
     'https://support.theguardian.com/contribute/climate-pledge-2019';
 
+const secondaryLinkUrl =
+    geolocation === 'US'
+        ? 'https://www.theguardian.com/environment/2019/oct/16/climate-crisis-america-guardian-editor-john-mulholland?acquisitionData=%7B%22source%22%3A%22GUARDIAN_WEB%22%2C%22componentType%22%3A%22ACQUISITIONS_EDITORIAL_LINK%22%2C%22campaignCode%22%3A%22climate_pledge_2019%22%2C%22componentId%22%3A%22climate_pledge_2019_banner%22%7D&INTCMP=climate_pledge_2019'
```

Comment:
interestingly the acquisitionData doesn't carry through to anywhere on www, it's a support. and subscribe. only parameter. The INTCMP does work on www though.

Reply:
that's really helpful to know!

Reply:
updated to remove the unused stuff